src/content/tutorials/observability/multi-tenancy/creating-grafana-organization/_index.md

```diff
@@ -91,41 +55,122 @@ spec:
+kind: GrafanaOrganization
+metadata:
+  name: engineering-team
+spec:
+  displayName: Engineering Team
+  rbac:
+    admins:
+    - customer:engineering-leads
+    editors:
+    - customer:senior-engineers
+    - customer:devops-team
+    viewers:
+    - customer:junior-engineers
+    - customer:qa-team
+  tenants:
+  - prod-frontend
+  - prod-backend
+  - staging-frontend
+  - staging-backend
+```
+
+**Production-only organization:**
 
-- **Grafana Organization**: A new organization named "MyOnlineShop" in Grafana
-- **Data Access**: Access to observability data stored within the `myonlineshop` tenant
-- **User Permissions**: Role assignments based on your identity provider groups
-- **Datasources**: Automatic provisioning of Loki, Mimir, and Alertmanager datasources filtered to the `myonlineshop` tenant
+This example shows a restricted organization with access only to production data:
+
+```yaml
+apiVersion: observability.giantswarm.io/v1alpha1
+kind: GrafanaOrganization
+metadata:
+  name: production-monitoring
+spec:
+  displayName: Production Monitoring
+  rbac:
+    admins:
+    - customer:sre-team
+    - customer:platform-admin
+    viewers:
+    - customer:engineering-team
+    - customer:support-team
+  tenants:
+  - production
+```
 
 ## RBAC configuration
 
-The Role Based Access Control (RBAC) section defines how to assign groups from your configured identity provider to `Grafana` [organization roles](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#organization-roles) (`Admin`, `Editor`, `Viewer`).
+The RBAC section maps identity provider groups to [Grafana organization roles](https://grafana.com/docs/grafana/latest/administration/roles-and-permissions/#organization-roles):
 
-### Dex connector format
+| Role | Permissions |
+|------|-------------|
+| **Admin** | Full organization access: manage users, datasources, dashboards, and settings |
+| **Editor** | Create and edit dashboards, alerts, and folders (cannot manage users) |
+| **Viewer** | Read-only access to dashboards and data |
+
+### Group format
+
+Most Giant Swarm installations use Dex as the identity provider. Specify groups using the format `{dex-connector-id}:{group-name}`:
+
+```yaml
+rbac:
+  admins:
+  - customer:platform-admin        # Maps 'platform-admin' group from 'customer' connector
+  - customer:ops-team
+  editors:
+  - customer:development-team
+  viewers:
+  - customer:support-team
+```
 
-Most Giant Swarm installations use Dex as the identity provider. When using Dex, specify groups using the format `{dex-connector-id}:{group-name}`. For example:
+**Finding your connector ID:** Check your cluster's Dex configuration for the connector ID (usually `customer`).
 
-- `customer:platform-admin` - Maps the `platform-admin` group from the `customer` Dex connector to Grafana Admin role
-- `customer:development-team` - Maps the `development-team` group from the `customer` Dex connector to Grafana Editor role
```

Comment:
Please keep the section on dex connector groups mapping.

Is there a doc to link about dex ? To explain where to find/configure the dex connector and also where are groups configured.

Reply:
Well if we have no doc we can't link it then :shrug:

Reply:
Well if we have no doc we can't link it then :shrug: